CONTRIBUTING.md

```diff
@@ -18,47 +18,46 @@ $ npm run bootstrap
 ## Development tasks
 
 ```sh
-$ npm run build # Build all the packages using Babel
-$ npm run test # Run the tests and linter a single time
-$ npm run test -- --package cf-component-button # Run the tests on a single package
-$ npm run test -- --no-instanbul # Turn off code coverage
-$ npm run test -- --browsers Chrome,Firefox # Specify the browsers you want to test in
-$ npm run dev # Run the tests and linter continuously (this support the same flags as test)
-$ npm run lint # Run the linter a single time
+npm start # Start the dev server, open http://localhost:8000 and start deving
+npm run build # Build all the packages using Babel (targets: commonjs, ES6) and Webpack (umd)
+npm run lint # Run the eslint
+npm run prettier # Reformats the whole JS codebase, look for an on-save integration for your editor
+npm test # Run the tests
+npm test -- --testPathPattern cf-component-button # Run the tests on a single package
+npm test -- --watchAll # Run the tests in watch mode
+npm run gen # Autogenerate the style guide (imports all components into one page)
 ```
 
-## Component examples
+## Style Guide
 
-To build the examples:
+### To build
 
 ```sh
-$ npm run examples:build:internal # Build the example site with CloudFlare's internal styles (employees only)
-$ npm run examples:build:external # Build the example site with CloudFlare's external styles
-$ open docs/index.html
+npm run build:styleguide
+open docs/index.html
 ```
 
-Or if you want to continuously work on the examples:
+### To work continuously
 
 ```sh
-$ npm run examples:dev:internal # Dev on the example site with CloudFlare's internal styles (employees only)
-$ npm run examples:dev:external # Dev on the example site with CloudFlare's external styles
+npm start
+open http://localhost:8000
 ```
 
+### To publish
+
+```sh
+npm run build:styleguide
+```
+
+and commit the changed `/docs` folder.
+
 ## Publishing packages
 
 cf-ui uses [Lerna](https://lernajs.io) in order to publish new versions of
 packages.
 
-If you were to manually publish everything it would essentially be:
 
 ```sh
-$ git checkout master
-$ git pull origin master --rebase
-$ ./node_modules/.bin/check-installed-dependencies
-$ npm run build
-$ cd packages/[[package-name]]
-# edit version inside package.json
-$ git tag -a "[[version]]"
-$ npm publish
-$ git push origin master --follow-tags
+npm run publish
```

Comment:
if someone wants to publish single/multiple packages is the 'npm run publish' will be used in both cases?